spec/bucket_store/disk_spec.rb

```diff
@@ -112,11 +111,11 @@
 
     context "when the bucket has some keys in it" do
       before do
-        instance.upload!(bucket: bucket, key: "2019-01/hello1", content: "world")
-        instance.upload!(bucket: bucket, key: "2019-01/hello2", content: "world")
-        instance.upload!(bucket: bucket, key: "2019-01/hello3", content: "world")
-        instance.upload!(bucket: bucket, key: "2019-02/hello", content: "world")
-        instance.upload!(bucket: bucket, key: "2019-03/hello", content: "world")
+        instance.upload!(bucket: bucket, key: "2019-01/hello1", file: file)
+        instance.upload!(bucket: bucket, key: "2019-01/hello2", file: file)
```

Comment:
How does this work? I would have guessed that we need a `.rewind` here? 🤔

Reply:
🤔 so I think this is working because it's creating a bunch of empty files within the backing store and the spec itself isn't verifying contents: it's just counting numbers of files. 

Can look to fix it up

Reply:
Fixed this